### PR TITLE
File server

### PR DESCRIPTION
**Pull request by RazanTayyem**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,6 +11,8 @@ pids
 *.seed
 *.pid.lock
 
+package-lock.json
```
Comment (jema28) [inline .gitignore]: This shouldn't be git ignored 😺

File: package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "mo-raz",
+  "version": "1.0.0",
+  "description": "## User Stories *As a FACK1 student...* > I can leave a review/feedback about the course",
```
Comment (jema28) [inline package.json]: Maybe change this to be a description about the app.

File: src/server/server.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const http = require('http');
+const router = require('./router.js');
+
+const PORT = 8001;
```
Comment (jema28) [inline src/server/server.js]: You could add a process.env.PORT here e.g. `const PORT  = process.env.PORT || 8001` 👍

**Comment by jema28** [comment]
Great work on keeping the PR small! Just a few small changes and then ready to merge ✨

File: .gitignore (hunk 1 above)
Comment (RazanTayyem) [reply .gitignore]: will done :smile:

File: package.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  "homepage": "https://github.com/FACK1/mo-raz#readme",
+  "devDependencies": {
+    "tap-spec": "^5.0.0",
+    "tape": "^4.9.1"
+  },
+  "dependencies": {
+    "dotenv": "^6.1.0",
+    "nodemon": "^1.18.7",
+    "pg": "^7.7.1",
+    "supertest": "^3.3.0"
```
Comment (jema28) [inline package.json]: Nodemon and supertest should be in dev dependencies 😸

Comment (RazanTayyem) [reply package.json]: Done

**Pull request merged**